app/controllers/master_files_controller.rb

```diff
@@ -47,6 +47,37 @@ def embed
     end
   end
 
+  def oembed
+    if params[:url].present?
+      pid = params[:url].split('?')[0].split('/').last
+      mf = MasterFile.find(pid)
+      if mf.present?
+        width = params[:maxwidth] || EMBED_SIZE[:medium]
```

Comment:
@bkeese EMBED_SIZE isn't defined in the controller context and is throwing an exception